server.js

```diff
@@ -132,7 +136,13 @@ class StaticmanAPI {
 
   requireService (services) {
     return (req, res, next) => {
-      const serviceMatch = services.some(service => service === req.params.service)
+      const serviceMatch = services.some(service => {
+        let requestedService = req.params.service
+        if (typeof requestedService === 'undefined') {
+          requestedService = ''
+        }
+        return service === requestedService
+      })
```

Comment:
```suggestion
      const serviceMatch = services.some(service => {
        let requestedService = req.params.service ? req.params.service : ''
        return service === requestedService
      })
```

I think we can use a ternary operator here for conciseness.

Reply:
Changed as requested.